sympy/codegen/rewriting.py

```diff
@@ -64,14 +64,20 @@ def __init__(self, cost_function=None, priority=1):
 class ReplaceOptim(Optimization):
     """ Rewriting optimization calling replace on expressions.
 
+    Explanation
+    ===========
+
     The instance can be used as a function on expressions for which
     it will apply the ``replace`` method (see
     :meth:`sympy.core.basic.Basic.replace`).
 
     Parameters
     ==========
-    query : first argument passed to replace
-    value : second argument passed to replace
+
+    query :
+        First argument passed to replace.
+    value :
+        Second argument passed to replace.
```

Comment:
Should this not say something about the types of these arguments?

Reply:
@aktech @asmeurer @Sc0rpi0n101 Could you look at this and suggest possible descriptions for the parameters?

Reply:
Ideally it should, but we don't have it mentioned in all of the docstrings and its not mandatory, we can improve that later as well, this is an improvement from style guide point of view anyways. I am happy to merge and improve types later.